peach.md

```diff
@@ -246,6 +246,8 @@ Many Bitcoiners prioritise Free and Open Source Software (FOSS) when selecting t
 ### Will Peach integrate Tor?
 At the moment, Tor integration is parked in their pipeline but not a priority.
 
+Because of that, a user is well advised to be on a [VPN](https://en.wikipedia.org/wiki/VPN_service) connection while interacting with the Peach app. Using a VPN is generally good practice on mobile phones and desktop devices. This way, the actual network IP of your device will not be visible to the services you connect to, thus gaining you network privacy, beneficial for all sorts of reasons. Fortunately, with providers like Mullvad VPN, for example, it's becoming more and more easy to use a good VPN even on mobile. It's not free though, it's a fiver a month – but it's worth it as soon as you learn to value your privacy, especially while making sensitive financial transactions. Big privacy bonus: you can even pay Mullvad via cash by mail (yes it works!) or in BTC.
```

Comment:
Thanks for creating this PR so fast! It is a very clever addition to the guide. 

Personally, I would keep it short and cut at "even on mobile" and link the episode of journey to sovereignty about VPN if the reader wants to learn more about the subject. 

What do you think? Feel free to push back 🙌

The best feedback will likely come from QnA 🤫

Reply:
I agree here. Let's remember that the focus here is Peach and letting the know that VPNs are a best practice. The existing hyperlink can explain more on what a VPN is.

Only thing I would add to update the sentence `Using a VPN is generally good practice on mobile phones and desktop devices.` to `Using a VPN like [Mullvad](https://mullvad.net) is generally good practice on mobile phones and desktop devices.`